v0/pipeline/steps/cd/helm-blue-green-stage-step-info.yaml

```diff
@@ -0,0 +1,54 @@
+title: HelmBlueGreenStepInfo
+allOf:
+  - $ref: ../../common/step-spec-type.yaml
+  - type: object
+    properties:
+      delegateSelectors:
+        oneOf:
+          - type: array
+            items:
+              type: string
+          - type: string
+            pattern: (<\+.+>.*)
+            minLength: 1
+      pruningEnabled:
```

Comment:
We don't need pruningEnabled, skipDeploymentIfSameManifest, skipDryRun. Instead add the fields from Helm Deployment: skip steady state check and ignore failed history